crates/chain/tests/test_tx_graph.rs

```diff
@@ -959,10 +959,10 @@ fn test_chain_spends() {
         }))
     );
 
-    // Even if unconfirmed tx has a last_seen of 0, it can still be part of a chain spend.
+    // Even if unconfirmed tx has a last_seen of None, it can still be part of a chain spend.
```

Comment:
Is this a valid assumption?

If instead we were to filter unseen txs in `try_get_chain_position`, it has the same desired effect on canonical txs, but then this assertion no longer holds.